processor/schemaprocessor/internal/translation/revision_v1.go

```diff
@@ -87,23 +81,28 @@ func newSpanChangeList(spans ast.Spans) *changelist.ChangeList {
 	return &changelist.ChangeList{Migrators: values}
 }
 
-func newMetricChangeList(metrics ast.Metrics) *changelist.ChangeList {
+func newMetricChangeList(metrics ast11.Metrics) *changelist.ChangeList {
 	values := make([]migrate.Migrator, 0)
 	for _, at := range metrics.Changes {
 		if renameAttributes := at.RenameAttributes; renameAttributes != nil {
 			attributeChangeSet := transformer.MetricDataPointAttributes{
 				ConditionalAttributeChange: migrate.NewConditionalAttributeSet(renameAttributes.AttributeMap, renameAttributes.ApplyToMetrics...),
 			}
 			values = append(values, attributeChangeSet)
-		} else if renamedMetrics := at.RenameMetrics; renamedMetrics != nil {
+		}
+		if renamedMetrics := at.RenameMetrics; renamedMetrics != nil {
 			signalNameChange := transformer.MetricSignalNameChange{SignalNameChange: migrate.NewSignalNameChange(renamedMetrics)}
 			values = append(values, signalNameChange)
 		}
+		if splitMetrics := at.Split; splitMetrics != nil {
+			// TODO: Implement split
```

Comment:
Probably worth adding an issue for this TODO since this is the primary difference that version 1.1 brings.

Side note: I think we need "new since <version>" at https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/schemas/file_format_v1.1.0.md to make more obvious what features were added in what version.

Reply:
Created https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/38402 . 
Agree we should add the "new since" section. I had manually diff the files to determine the delta.